index.js

```diff
@@ -0,0 +1,65 @@
+let booksCollection = [];
+
+const setBooks = (books) => {
+  localStorage.setItem('Books', JSON.stringify(books));
+};
+
+const getBooks = () => {
+  const books = JSON.parse(localStorage.getItem('Books'));
+  if (books) return books;
+  return [];
+};
+
+const updatedBooksStorge = (book) => {
+  booksCollection = getBooks();
+  booksCollection.push(book);
+  setBooks(booksCollection);
+};
+
+const displayBook = () => {
+  const books = getBooks();
+  const booksContainer = document.querySelector('.books');
+  booksContainer.innerHTML = '';
+  for (let i = 0; i < books.length; i += 1) {
+    booksContainer.innerHTML += `
+        <div class="book">
+        <p>${books[i].title}</p>
+        <p>${books[i].autor}</p>
```

Comment:
- [ ] When I add a book it only displays the title, but not the author because of your error here. We should have `author`, not `autor`

  ![image](https://user-images.githubusercontent.com/81468675/211421001-b0b7a529-17a1-4aa3-80cc-eed7a8764413.png)

Reply:
well resolved! ✔️